taxi/urls.py

```diff
@@ -37,14 +42,65 @@
         ManufacturerDeleteView.as_view(),
         name="manufacturer-delete",
     ),
-    path("cars/", CarListView.as_view(), name="car-list"),
-    path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
-    path("cars/create/", CarCreateView.as_view(), name="car-create"),
-    path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
-    path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
-    path("drivers/", DriverListView.as_view(), name="driver-list"),
     path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
+        "cars/",
+        CarListView.as_view(),
+        name="car-list"
+    ),
+    path(
+        "cars/<int:pk>/",
+        CarDetailView.as_view(),
+        name="car-detail"
+    ),
+    path(
+        "cars/<int:pk>/driver-add/",
+        add_driver_to_car,
+        name="driver-add"
+    ),
+    path(
+        "cars/<int:pk>/driver-remove/",
+        remove_driver_from_car,
```

Comment:
Ensure that the `remove_driver_from_car` view has proper permission checks to prevent unauthorized access, as highlighted in the previous review.